ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -9,12 +9,6 @@
 
 package com.facebook.react.modules.network;
 
-import javax.annotation.Nullable;
-
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.Reader;
-
```

Comment:
Could you make sure the imports doesn't move here. There should be an option in your IDE. The problem is that another PR to this file will probably end moving them back destroying history again.

Same of the other file.